HelpMyStreet.Utils/HelpMyStreet.Utils/Models/User.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HelpMyStreet.Utils.Models
+{
+    public class User
+    {
+        public int ID { get; set; }
+        public string FirebaseUID { get; set; }
+        public string PostalCode { get; set; }
+        public bool EmailSharingConsent { get; set; }
```

Comment:
These are nullable in the database. Is that correct

Reply:
Yes